packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/bedrock/create-model-customization-job.ts

```diff
@@ -0,0 +1,428 @@
+  }
+
+  /**
+   * Configure the IAM role for the bedrock create model customization job
+   *
+   * @see https://docs.aws.amazon.com/bedrock/latest/userguide/model-customization-code-samples.html
+   */
+  private renderBedrockCreateModelCustomizationJobRole(): iam.IRole {
+    if (this.props.role) {
+      return this.props.role;
+    }
+    const stack = Stack.of(this);
+    const role = new iam.Role(this, 'BedrockRole', {
+      assumedBy: new iam.ServicePrincipal('bedrock.amazonaws.com'),
+      inlinePolicies: {
+        BedrockCreateModelCustomizationJob: new iam.PolicyDocument({
+          statements: [
+            ...(this.props.vpcConfig ? [
+              new iam.PolicyStatement({
+                actions: [
+                  'ec2:DescribeNetworkInterfaces',
+                  'ec2:DescribeVpcs',
+                  'ec2:DescribeDhcpOptions',
+                  'ec2:DescribeSubnets',
+                  'ec2:DescribeSecurityGroups',
+                ],
+                resources: ['*'],
+              }),
+              new iam.PolicyStatement({
+                actions: ['ec2:CreateNetworkInterface'],
+                resources: [
+                  stack.formatArn({
+                    service: 'ec2',
+                    resource: 'network-interface',
+                    resourceName: '*',
+                  }),
+                  stack.formatArn({
+                    service: 'ec2',
+                    resource: 'security-group',
+                    resourceName: '*',
+                  }),
+                  stack.formatArn({
+                    service: 'ec2',
+                    resource: 'subnet',
+                    resourceName: '*',
+                  }),
+                ],
+              }),
+              new iam.PolicyStatement({
+                actions: ['ec2:CreateTags'],
+                resources: [stack.formatArn({
+                  service: 'ec2',
+                  resource: 'network-interface',
+                  resourceName: '*',
+                })],
+                conditions: {
+                  StringEquals: {
+                    'ec2:CreateAction': 'CreateNetworkInterface',
+                  },
+                },
+              }),
+              new iam.PolicyStatement({
+                actions: [
+                  'ec2:CreateNetworkInterfacePermission',
+                  'ec2:DeleteNetworkInterface',
+                  'ec2:DeleteNetworkInterfacePermission',
+                ],
+                resources: ['*'],
+                conditions: {
+                  StringEquals: {
+                    'ec2:Subnet': [
+                      ...(this.props.vpcConfig
+                        ? this.props.vpcConfig.subnets.map((subnet) => subnet.subnetId)
+                        : []),
+                    ],
+                  },
+                },
+              }),
+            ] : []),
+            new iam.PolicyStatement({
+              actions: ['s3:GetObject'],
```

Comment:
From the linked policy, we also need `s3:ListBucket` for the training and validation buckets (and the required bucket-level resource).